x-pack/plugins/cases/server/telemetry/queries/pushed.test.ts

```diff
@@ -86,6 +86,7 @@ describe('pushes', () => {
         page: 0,
         perPage: 0,
         type: 'cases-user-actions',
+        namespaces: ['*'],
```

Comment:
Why is this test called `pushed.test.ts` 😄 I think it's a typo.

Reply:
lol I think it should be `push` you are right. I will change it.